### ISSUE TITLE
Many "Playback Error (undefined)" starting this week...

### ISSUE THREAD

**Issue by husterk** [Atraci/Atraci#256]
Most of the songs that I attempt to play end up with a "Playback Error (undefined)" alert box. Some still work properly but most don't. I have attached an image of the errors logged to the development console. I tried clearing the database and cache but neither helped. Seems that all of the failed songs are throwing video.js errors. I first noticed this earlier this week. I am currently running Atraci v0.7.0.

![videojs-errors](https://cloud.githubusercontent.com/assets/8089998/6047754/9a1d060e-ac79-11e4-8e15-30d2ca8e7ddb.png)

**Comment by shyne99** [comment]
Hello,

exactly the same errors here. I've just downloaded it to my PC, and almost only "Playback Error (undefined)". The log console throw the same errors on video.js.... I run the last version of atraci v0.7.0.

**Comment by katewarne** [comment]
Same errors on both Mac and Linux. Running latest version. On linux I uninstalled, deleted the hidden folders and reinstalled. Same errors.

**Comment by gonzalezb** [comment]
Same here almost every song i have realized though songs with lyrics play fine but they can not be vevo versions.

**Comment by shyne99** [comment]
Hello,

Can we have a status about this issues, did the source of the problem have been at least found ?? 

I'm enjoying my android version without any problem, and I never had a functional desktop version since this playback error occur on 80% of the songs that I try to play.

Thanks a lot for your work guys.

**Comment by fa7ad** [comment]
Could you guys try building from my fork? For some reason lot of you are facing issues but I'm using ti right now and all the songs seem to be playing fine :confused:

**Comment by shyne99** [comment]
Hello fa7ad, can you tell us which version of node-webkit aren you using in your functionnal version ?

**Comment by fa7ad** [comment]
The latest one. 0.12-alpha2 i think. 
See my pull request for the correct version (i'm on my phone atm)

**Comment by fa7ad** [comment]
I dont think a lot of bugs will be solved here anytime soon. The team is very busy working on Atraci/Atraci-angular

**Comment by shyne99** [comment]
Yeah that's what i understood, but i really love the idea, so, even with my little skills in IT, i am trying to solve the issue all by myself... I spent two hours just to compile it properly so, it will take a while, and i am really not sure that i will be able to do it myself... But the persons that never fails are the persons that never try.

**Comment by fa7ad** [comment]
Wel it works for me, i had the single frame video issue so i followed some comments on the issue and fixed it on my fork. I think this might be related to node-webkit as well. Thats why i suggested you try my fork :)

**Comment by avihaymenahem** [comment]
@shyne99 where you are from?
Youtube api calls several video providers from outside US

**Comment by shyne99** [comment]
I am from France. 

I think that the problem comes from the link that generate ytdl. Some are working here like this one http://r3---sn-25ge7nek.googlevideo.com/videoplayback?mm=31&pl=19&expire=1424113593&mt=1424091809&id=o-ABACKwp6iqAskZVsSUPObf4d6XuYG868rmTO5DsQftkV&dur=0.000&ip=80.11.131.52&initcwndbps=1837500&sver=3&mime=video/webm&source=youtube&key=yt5&itag=43&upn=MfGVd3PWewU&sparams=dur,id,initcwndbps,ip,ipbits,itag,mime,mm,ms,mv,pl,ratebypass,source,upn,expire&signature=832FD23C8F70EF9BC1FD0ED47F7EC36D66E20F4F.8807B5DBC7E7577669945EE275FDC86148AF9E49&ratebypass=yes&ipbits=0&mv=m&ms=au&fexp=900241,907263,913445,924638,927622,929305,931385,937412,9405661,9405892,943606,943917,947225,948124,952302,952605,952612,952901,955301,957201,959701

Other don't work at all like this one

http://r5---sn-25ge7n76.googlevideo.com/videoplayback?gcr=fr&fexp=907263,927622,930811,937407,9406007,9406421,9406778,9406848,943917,947225,948124,952302,952605,952612,952901,955301,957201,958600,959701&initcwndbps=1887500&mime=video/webm&key=yt5&ip=80.11.131.52&upn=i8xxFfmdUUg&itag=43&sparams=dur,gcr,id,initcwndbps,ip,ipbits,itag,mime,mm,ms,mv,pl,ratebypass,source,upn,expire&sver=3&source=youtube&ms=au&mt=1424091809&pl=19&ipbits=0&id=o-ACFp4__mfbSocPMEC97B01UmpjL9R6KwuW8M0vARQcc3&mm=31&expire=1424113591&mv=m&dur=0.000&ratebypass=yes

(Sorry i don't know how to reduce the lenghts of the links...)

**Comment by shyne99** [comment]
The problem really seems tpo come from ytdl. Using it in command line, the links that he provides me don't seems to work....

**Comment by shyne99** [comment]
The best solution would be to port this project to youtube-dl which seems to work fine for everyone. 

The problem is that we will have to install python with Atraci.... 

But all this are way over my skills...

**Comment by husterk** [comment]
@fa7ad - Would you be able to provide a built version (for Windows) of your fork so that I can test it?

**Comment by fa7ad** [comment]
@husterk I use linux and I think I deleted my fork. Bsically, update node-webkit to the latest version and tweak with the ytdl version a little

**Comment by fa7ad** [comment]
@husterk also, try pulling from the master branch again since @mimi89999 added a possible fix for this

**Comment by husterk** [comment]
@fa7ad - Thanks for the tip but it still isn't working. I downloaded the latest master and built it using Node-Webkit v0.12.1 for WinX64 but now none of the songs play. The albums load when you perform a search but when you click on an album you just see the loading spinner and no error messages in the debug console. I will try to look at this further later this week.

**Comment by fa7ad** [comment]
yeah, mine stopped working today as well. Probably something to do with the API. @avihaymenahem is the API down?

**Comment by mimi89999** [comment]
Yes, youtube stoped support for api v2 :( .

**Comment by 5punk** [comment]
@mimi89999 @fa7ad @husterk @shyne99 @avihaymenahem @katewarne 

Please continue here.
https://github.com/Atraci/Atraci/issues/265
